src/qt/bitcoingui.cpp

```diff
@@ -973,7 +973,7 @@ void BitcoinGUI::createActions()
 	votingAction->setMenuRole(QAction::TextHeuristicRole);
 
 
-	votingReservedAction = new QAction(QIcon(":/icons/bitcoin"), tr("&Reserved - Voting Linux"), this);
+    votingReservedAction = new QAction(QIcon(":/icons/bitcoin"), tr("&Voting Linux"), this);
```

Comment:
Yes, I agree. Didn't change that because I didn't know what to do with the old voting dialogs. Should we remove them or keep them for the windows users (at least for a while)?